### PR TITLE
[Element 2.0] Landing page for documentation of Element 2.0

### PR DESCRIPTION
**Pull request by honglax**
Resolve [https://tricentisflood.atlassian.net/browse/FLO-2132](https://tricentisflood.atlassian.net/browse/FLO-2132)

### CODE REVIEW

File: packages/docs/src/components/v2LandingPage/index.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+						Stay up-to-date on how your test is executed in real-time with the intuitive and
+						progressive terminal output.
+					</li>
+					<li>
+						Find what you're looking for at first glance thanks to smart indentation, spacing and
+						coloring.
+					</li>
+					<li>View either concise or comprehensive output with the optional verbose mode.</li>
+					<li>
+						See taken screenshots right within the terminal <i>(only on iTerm2 - MacOS).</i>
```
Comment (HieuLeKMS) [inline packages/docs/src/components/v2LandingPage/index.js:26]: It's `iTerm` in the source code & the doc. Then which is correct? iTerm or iTerm2

Comment (honglax) [reply packages/docs/src/components/v2LandingPage/index.js:26]: iTerm2 is correct.

Comment (honglax) [reply packages/docs/src/components/v2LandingPage/index.js:26]: Hi a @phileba, please confirm it.

Comment (honglax) [reply packages/docs/src/components/v2LandingPage/index.js:26]: Resolved

**Review by HieuLeKMS — APPROVED** [review]

**Pull request merged**